### PR TITLE
Added edited node properties to state using hooks

### PR DESCRIPTION
**Pull request by nehashende**

### CODE REVIEW

File: src/browser/modules/EditorInfo/DisplayNodeDetails.jsx [hunk 1]
```diff
@@ -105,56 +105,107 @@ EntitySection.propTypes = {
...
-                      )
-                    }}
-                  />
-                </StyledValue>
-              </tr>
-            </tbody>
-          </StyledTable>
-        </div>
-      )
+export class PropertiesSection extends Component {
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/DisplayNodeDetails.jsx]: use recompose for managing component state

**Comment by rahul-tarafdar-synerzip** [comment]
Use hooks and keep functional components

**Comment by rahul-tarafdar-synerzip** [comment]
also resolve conflicts

**Comment by nehashende** [comment]
> Use hooks and keep functional components

Added hooks and resolved conflicts

**Review by rahul-tarafdar-synerzip — APPROVED** [review]

**Pull request merged**